### PR TITLE
chore(notify-register): replace enableSync by register

### PR DESCRIPTION
**Pull request by Cali93**

### CODE REVIEW

File: src/contexts/W3iContext/hooks/pushHooks.ts [hunk 1]
```diff
@@ -47,12 +47,13 @@ export const usePushState = (w3iProxy: Web3InboxProxy, proxyReady: boolean, dapp
 
   const handleRegistration = useCallback(
     async (key: string) => {
+      console.log({ pushClient, key, pushEnabled: uiEnabled.notify })
```
Comment (devceline) [inline src/contexts/W3iContext/hooks/pushHooks.ts:50]: Do we need this log?

File: src/contexts/W3iContext/hooks/pushHooks.ts [hunk 2]
```diff
@@ -47,12 +47,13 @@ export const usePushState = (w3iProxy: Web3InboxProxy, proxyReady: boolean, dapp
...
       if (pushClient && key && uiEnabled.notify) {
         try {
           const identityKey = await pushClient.register({ account: `eip155:1:${key}` })
+          refreshPushState()
           setRegisterMessage(null)
           setRegistered(identityKey)
-          refreshPushState()
```
Comment (devceline) [inline src/contexts/W3iContext/hooks/pushHooks.ts:53]: What's the reasoning for this?

**Review by devceline — APPROVED** [review]
LGTM

File: src/contexts/W3iContext/hooks/pushHooks.ts (hunk 2 above)
Comment (Cali93) [reply src/contexts/W3iContext/hooks/pushHooks.ts:53]: I looked into how `chatHooks` handle it and I used the same order, curious if it matters ?

**Pull request merged**